dsa-solutions/gfg-solutions/Hard/0101-0200/0112-Fixing Two swapped-nodes-of-a-BST.md

```diff
@@ -0,0 +1,173 @@
+---
+id: Fixing Two swapped nodes of a BST
```

Comment:
replace `id: Fixing Two swapped nodes of a BST` to `id: fixing-two-swapped-nodes-of-a-bst`